app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/apiservices/PokemonService.kt

```diff
@@ -0,0 +1,21 @@
+package au.com.gridstone.trainingkotlin.apiservices
+
+import au.com.gridstone.trainingkotlin.data.PokemonResults
+import au.com.gridstone.trainingkotlin.data.PokemonData
+import retrofit2.Response
+import retrofit2.http.*
+
+interface PokemonService {
+  @GET("?limit=151")
+  suspend fun getPokemons(
+//    @Url url: String = "https://httpstat.us/400"
+  ): Response<PokemonResults>
+
+  @GET("{pokemonUrl}")
+  suspend fun getPokemonDetails(@Path("url") url: String): Response<PokemonData>
+}
+
+
+//  @POST("parts") fun addPartAsync(@Body newPart : PartData): Deferred<Response<Void>>
+//  @DELETE("parts/{id}") fun deletePartAsync(@Path("id") id: Long) : Deferred<Response<Void>>
+//  @PUT("parts/{id}") fun updatePartAsync(@Path("id") id: Long, @Body newPart: PartData) : Deferred<Response<Void>>
```

Comment:
no need to commit in commented out code 😄

Reply:
sure keep in mind next time